src/main/java/com/sprint/mission/discodeit/message/controller/BinaryContentController.java

```diff
@@ -0,0 +1,43 @@
+package com.sprint.mission.discodeit.message.controller;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.sprint.mission.discodeit.binaryContent.entity.BinaryContent;
+import com.sprint.mission.discodeit.binaryContent.service.BinaryContentService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/binaryContents")
```

Comment:
url은 보통 카멜케이스보단 `-`(하이픈)을 사용해요.  
```suggestion
@RequestMapping("/api/binary-contents")
```